### ISSUE TITLE
Can you customize action/condition list?

### ISSUE THREAD

**Issue by Cestis** [steffendx/GoNorth#117]
Is it possible to remove certain actions/conditions seen in their select menus? Ideally, there would be an option to create custom elements, but the most important thing is probably to remove redundant ones (for example, in games where NPCs don't have their routines, seeing that option could be confusing for quest makers).

**Comment by steffendx** [comment]
Good idea. I guess I could fit something like this into the project settings area. That way one could enable and disable certain features like this.
